CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Please refer to [UPGRADING](UPGRADING.md) guide for upgrading to a major version.
 
+## 7.3.0 - 2021-03-21
+
+### Added
+
+- Support for DER and P12 certificates [#2413](https://github.com/guzzle/guzzle/pull/2413(
+- Support the cURL (http://) scheme for StreamHandler proxies [#2850](https://github.com/guzzle/guzzle/pull/2850)
+- Support for `guzzlehttp/psr7:2.0` [#2878](https://github.com/guzzle/guzzle/pull/2878)
```

Comment:
Maybe call it `^2.0`. Don't want people trying to copy this as a version constraint.

Reply:
```suggestion
- Support for `guzzlehttp/psr7:^2.0` [#2878](https://github.com/guzzle/guzzle/pull/2878)
```

Sure!